### PR TITLE
Align deployment rollback link right

### PR DESCRIPTION
**Pull request by philipnrmn**
Before:

After:

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @orlandohohmeier

File: src/js/pages/services/DeploymentsTab.js [hunk 1]
```diff
@@ -228,7 +228,7 @@ class DeploymentsTab extends mixin(StoreMixin) {
         linkText = linkText + ' & Remove';
       }
       return (
-        <a className="deployment-rollback button button-link button-danger table-display-on-row-hover"
+        <a className="deployment-rollback button button-link button-danger table-display-on-row-hover pull-right"
```
Comment (mlunoe) [inline src/js/pages/services/DeploymentsTab.js]: I wasn't aware of this class, but I believe it is legacy. I think we should stay consistent with what we do other tables and use `text-aling-right` on the `td`, like here: https://github.com/dcos/dcos-ui/blob/master/src/js/components/PackagesTable.js#L51

Comment (jfurrow) [inline src/js/pages/services/DeploymentsTab.js]: Agreed, `pull-right` does exist in Canvas, but it makes use of `float: right;` which I am not a fan of. Though you could argue this is personal preference...

**Comment by orlandohohmeier** [comment]
LGTM 💯 ✨

**Pull request merged**